### PR TITLE
add css-classes for next, previous, first and last pager links

### PR DESCRIPTION
**Pull request by siyamandayubi**

### CODE REVIEW

File: src/OrchardCore.Modules/OrchardCore.Navigation/PagerShapes.cs [hunk 1]
```diff
@@ -360,6 +360,7 @@ public Task<IHtmlContent> Pager_First(Shape Shape, dynamic DisplayAsync)
         {
             Shape.Metadata.Alternates.Clear();
             Shape.Metadata.Type = "Pager_Link";
+            ((dynamic)Shape).Tag.AddCssClass("first");
```
Comment (sebastienros) [inline src/OrchardCore.Modules/OrchardCore.Navigation/PagerShapes.cs]: Wrong approach, use Shape.AddCssClass instead and handle them in the templates

**Comment by Skrypt** [comment]
I don't see why we would require this.
I'm looking at the HTML without this changes and the <li> element already has a class="last" assigned for example. The CoreShapes.cs file appends already this class to any element that would be first or last. The only thing it adds is redundancy on the contained <a> anchor.

image

Example show in this screenshot where "first" is done with `Shape.Classes.Add("first");` and we can see that this class is already added on it's parent <li> element. Second example is without on the "last" element.

**Comment by hishamco** [comment]
@Skrypt shall we close this one?

**Comment by Skrypt** [comment]
Yes

**Pull request closed without merging**